notebooks/en/issues_in_text_dataset.ipynb

```diff
@@ -0,0 +1,3635 @@
+{
```

Comment:
<p>The doc-builder that we use to publish notebooks, seems to have issues with the <code>&lt;div&gt;</code>  tags in this markdown. Please reformat to remove them, and leave only the markdown formatting. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/huggingface/cookbook/pull/30/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>